Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -3466,7 +3466,7 @@ - (BOOL)boolNamed:(NSString *)name
 {
     if ([Utils isNullOrEmpty:name]) {
         [Leanplum throwError:@"[LPActionContext boolNamed:] Empty name parameter provided."];
-        return nil;
+        return NO;
```

Comment:
not sure if we can make this change? it will be breaking... prob. need to double check with andrew, also we should make sure android is aligned